MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,893 @@
+<td>{{'Content'| markdownify }}</td>
+<td>Gets or sets a any view to display in the center of circular progress bar.</td></tr>
+<tr>
+<td>{{'[IndicatorInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorInnerRadius)'| markdownify }}</td>
+<td>{{'ProgressThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness of the progress. You can specify the value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
+<tr>
+<td>{{'[IndicatorOuterRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorOuterRadius)'| markdownify }}</td>
+<td>{{'ProgressRadiusFactor'| markdownify }}</td>
+<td>Gets or sets a value that specifies the outer radius factor of the progress.</td></tr>
+<tr>
+<td>{{'[TrackInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_TrackInnerRadius)'| markdownify }}</td>
+<td>{{'TrackThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness of the track in the circular progress bar. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
+<tr>
+<td>{{'[TrackOuterRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_TrackOuterRadius)'| markdownify }}</td>
+<td>{{'TrackRadiusFactor'| markdownify }}</td>
+<td>Gets or sets a value that specifies the outer radius factor of the track.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'ThicknessUnit'| markdownify }}</td>
+<td>Gets or sets the enum value that indicates to calculate the track and progress thickness either in logical pixel or radius factor.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'ProgressCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the progress.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'TrackCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the track.</td></tr>
+<tr>
+<td>{{'[ShowProgressValue](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue)'| markdownify }}</td>
+<td>{{'-'| markdownify }}</td>
+<td>Gets or sets a value indicating whether to show the progress value label or not in SfCircularProgressBar.</td></tr>
+
+</table> 
+
+The following code example explains how to initialize the Xamarin SfCircularProgressBar and the .NET MAUI SfCircularProgressBar along with its properties.
```

Comment:
The following code example explains how to initialize the Xamarin SfCircularProgressBar and the .NET MAUI SfCircularProgressBar along with its properties: